dev/provider_packages/prepare_provider_packages.py

```diff
@@ -1318,6 +1327,25 @@ def update_release_notes(
     return True
 
 
+def update_min_airflow_version(provider_package_id: str, version_suffix: str):
+    provider_details = get_provider_details(provider_package_id)
+    provider_info = get_provider_info_from_provider_yaml(provider_package_id)
+    jinja_context = get_provider_jinja_context(
+        provider_info=provider_info,
+        provider_details=provider_details,
+        current_release_version=provider_details.versions[0],
+        version_suffix=version_suffix,
+    )
+    generate_init_py_file_for_provider(
+        context=jinja_context,
+        target_path=provider_details.source_provider_package_path,
+    )
+    if not jinja_context["PREINSTALLED_PROVIDER"]:
+        replace_min_airflow_version_in_provider_yaml(
+            context=jinja_context, target_path=provider_details.source_provider_package_path
+        )
```

Comment:
Where is the part that skip this for providers in the version exception list?
We don't want min version 2.6 to be override with 2.5

Reply:
```
    min_airflow_version = MIN_AIRFLOW_VERSION
    if MIN_AIRFLOW_VERSION_EXCEPTIONS.get(provider_details.provider_package_id):
        min_airflow_version = MIN_AIRFLOW_VERSION_EXCEPTIONS[provider_details.provider_package_id]
```